### ISSUE TITLE
Look through “OpenStreetMap World” Discord

### ISSUE THREAD

**Issue by danieldegroot2** [osmlab/awesome-openstreetmap#62]
**Link To Project**
See [Global communities](https://github.com/osmlab/awesome-openstreetmap#global-communities)

**Short description of the Project**
In the channel list on the left hand side you will find a channel with useful links called `#tools`.

**Why is it awesome?**
It's another kind of curated list of OpenStreetMap projects.

**Additional Links**
See related wiki pages where available or search using the browser of your preference

**Comments**
Have fun! ;-)

**Comment by Unkn0wnCat** [comment]
You're right, there will probably be some awesome projects in there we don't have yet! 👍🏼 

*If you (as in the one reading this) would like to do this, please leave a comment below so I can assign this issue to you!*
